### ISSUE TITLE
Deprecate chunks keyword in write_image` and write_image_multiscales

### ISSUE THREAD

**Issue by sbesson** [ome/ome-zarr-py#167]
See conversation at https://github.com/ome/ome-zarr-py/pull/161#discussion_r800575583

The `storage_options` keyword introduced in 0.3.0 should support setting the `chunks` to the Zarr dataset. This option also gives the extra flexibility of being able to set different chunk sizes for each resolution. The explicit `chunks` keyword could be deprecated in an upcoming `0.x.0` release of the `ome-zarr` library.

**Comment by constantinpape** [comment]
I agree: `chunks` should be deprecated in favor of `storage_options`.

**Comment by will-moore** [comment]
FYI - thoughts on moving `storage_options` to `parse_url()` instead of `write_multiscale()` - which would mean that we keep `chunks` keyword - see #282
